playbooks/maas-agent-setup.yml

```diff
@@ -73,29 +73,6 @@
         mode: "0755"
 
   post_tasks:
-    - name: Assign agent ID to entity
```

Comment:
That still happens, it just got rearranged into `maas-token-setup.yml` since the poller also needs the same stuff.